### DISCUSSION TITLE
How to find the tex files that have not been saved？

### DISCUSSION THREAD

**Discussion by zhuofupan** [James-Yu/LaTeX-Workshop#3102] [Q&A]
I did not turn on the autosave function of VScode before.
Every time I compile, I can successfully generate a pdf, which causes me to forget to save the tex file for a long time.
So the tragedy happened.
When I turned on my computer today, the tex file reverted back to the editing state of a few days ago.
So, if there is an auto-saved tex file that I can find back?
Where might I be able to find it?

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

**Comment by zhuofupan** [comment]
Thanks. I've rewritten the missing ones. 
It should be difficult to find back.
Anyway, thank you.

**Comment by tamuratak** [comment]
Please open a new issue if any progress.

**Comment by jlelong** [comment]
This is strange as VS Code does regular backups of unsaved files, see https://github.com/Microsoft/vscode/issues/21183. You can also have a look at https://datarecoverypit.com/recover-vs-code-files/ Method 2.

I Hope this helps.

**Comment by James-Yu** [comment]
This question is not related to the extension, but VS code. A better place for help may be https://github.com/microsoft/vscode.
